app/views/posts/index.html.erb

```diff
@@ -1 +1,11 @@
-<h2> This is cool </h2>
\ No newline at end of file
+<section >
+  <%= render'partials/user', user: @user %>
+    <% @posts.each do |post| %>
+   <a href="/users/<%= @user.id %>/posts/<%= post.id %>">
+    <ul class="post-list">
+    <%= render 'partials/post', post: post %></ul>
+    </a>
```

Comment:
- [ ] Kindly maake sure you use `post.recent_five_comments` method to render **5 most recent comments** in the `app/views/posts/index.html.erb` page as required for the project. I noticed that in your `app/views/partials/_post.html.erb` partial file you render all the comments for a post. Make sure you fix this to match the requirement.

Reply:
@tresorsawasawa  I have read through the requirements over again and I can't find such a requirement

Reply:
Hi @DuaneDave 

If you check from this **[wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_user_all_posts.png)** and on images below, you will notice that:
![Screenshot from 2022-12-01 15-10-42](https://user-images.githubusercontent.com/83042742/205062635-af0ae5df-dffb-47d6-bf8b-339163aa5a84.png)
![Screenshot from 2022-12-01 15-14-28](https://user-images.githubusercontent.com/83042742/205062691-3da75961-64a5-43b9-8460-db55fd77e37a.png)

**In short, on each post you should render 5 recent comments**

Reply:
@tresorsawasawa I have added that method now